app/controllers/api/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2012 Brian Ploetz (bploetz@gmail.com)
```

Comment:
> There is no need to include licenses from Gems we use

Got it. This is clear indeed.

> superseded by own work

Depends on your definition here. I think in the sense that almost every line has been refactored/changed somehow, they have been "superseded".  However, I have heavily referenced Nov's work in the three folders I mentioned above. What do you think I should do?